template.ejs

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="UTF-8">
     <title><%= webpackConfig.template.title %></title>
+    <meta name="description" content="webpack is a module bundler. It packs CommonJs/AMD modules i. e. for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand.">
```

Comment:
* CommonJs/AMD modules → ES2015/CommonJS/AMD modules
* "i. e." does not make any sense here?
* Maybe also mention "loaders" for CSS, Images, JSON, LESS, …? (see project summary on https://github.com/webpack/webpack)